src/main/java/org/kainos/ea/resources/EmployeeController.java

```diff
@@ -42,6 +46,23 @@ public Response getAllEmployeesDeliveryEmployees () {
             return Response.serverError().build();
         }
     }
+
+    @POST
+    @Path("/employee/delivery")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response createNewEmployee (EmployeeRequest employeeRequest) {
+
+        try {
+            return Response.ok().entity(employeeService.createNewEmployee(employeeRequest)).build();
+        } catch (DeliveryEmployeeCouldNotBeCreatedException | EmployeeRequestIsNotValid e) {
```

Comment:
DeliveryEmployeeCouldNotBeCreatedException should be returning a 500

Reply:
opps - sorted now???